relay-server/src/actors/outcome.rs

```diff
@@ -197,6 +198,21 @@ impl Outcome {
     }
 }
 
+impl fmt::Display for Outcome {
```

Comment:
Since the start, we've used `{:?}` to serialize outcomes, even though that was never very readable. Now that we use outcomes more often in error logging and capture mode, it's worth adding this display implementation.